Makefile

```
@@ -1,5 +1,23 @@
 TARGETS=go1.16 go1.17
 ARM_TARGETS=go1.16 go1.17
+# TODO: to be replaced once we validate it works as expected
+# TODO: version to be tagged too.
+NPCAP_VERSION=1.60
+#NPCAP_FILE=npcap-$(NPCAP_VERSION)-oem.exe
+NPCAP_FILE=test.txt
```

TODO: to be replaced once we agree this is the way to go

```suggestion
NPCAP_VERSION=1.60
NPCAP_FILE=npcap-$(NPCAP_VERSION)-oem.exe
```